api/src/app.service.ts

```diff
@@ -47,6 +65,7 @@ export class AppService {
       gameState.currentPlayer,
       gameState.players,
     );
+    setGameState(newGameState);
```

Comment:
As `setGameState` return the new state, I would do that like this:

```js
return setGameState({
    ....gameState,
    graph: moveMarbleInDirection(graph, coordinates, direction),
    currentPlayer: switchToNextPlayer(
      gameState.currentPlayer,
      gameState.players,
    );
});
```

Reply:
Done properly